opentasks/src/main/java/org/dmfs/tasks/utils/MapOpener.java

```diff
@@ -0,0 +1,30 @@
+package org.dmfs.tasks.utils;
+
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+
+
+/**
+ * Handles opening map application on the device with a location query string.
+ *
+ * @author Gabor Keszthelyi
+ */
+public class MapOpener
```

Comment:
This class doesn't feel right and probably shouldn't exist. I suggest to make this a private method of the LocationFieldView.